pkg/ottl/expression_test.go

```diff
@@ -1550,3 +1551,66 @@ func Test_StandardPMapGetter_WrappedError(t *testing.T) {
 	_, ok := err.(TypeError)
 	assert.False(t, ok)
 }
+
+func Test_StandardTimeGetter(t *testing.T) {
```

Comment:
Please add a new test func for a `_WrappedError` scenario.

Reply:
Added!